### PR TITLE
relaties_Mohamed

### PR DESCRIPTION
**Pull request by medkaouchi**
relaties, ProductSupplier, ...

### CODE REVIEW

File: ClassLibrary3/Repository/AppDbContext.cs [hunk 1]
```diff
@@ -146,7 +146,116 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
...
+            //ProductReview-User
+            modelBuilder.Entity<ProductReview>()
+                .HasOne<User>(pr => pr.User)
+                .WithMany(u => u.ProductReviews)
+                .HasForeignKey(pr => pr.UserId);
+
+            // Tranporter-User
+            modelBuilder.Entity<Transporter>()
+                .HasOne<User>(t => t.User)
+                .WithMany(u => u.Tranporters)
```
Comment (ClaireWhitten) [inline ClassLibrary3/Repository/AppDbContext.cs]: .WITHONE(u=>u.Tran**S**porter)

File: ClassLibrary3/Repository/AppDbContext.cs [hunk 2]
```diff
@@ -146,7 +146,116 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
...
+            // Tranporter-Zone
+            modelBuilder.Entity<Zone>()
+                .HasOne<Transporter>(z => z.Transporter)
+                .WithMany(t => t.Zones)
+                .HasForeignKey(z => z.TransporterID);
+
+            //Adress-User
+            modelBuilder.Entity<Address>()
+                .HasOne<User>(a => a.User)
+                .WithMany(u => u.Adresses)
```
Comment (ClaireWhitten) [inline ClassLibrary3/Repository/AppDbContext.cs]: ADDresses

File: ClassLibrary3/Repository/AppDbContext.cs [hunk 3]
```diff
@@ -146,7 +146,116 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
...
+                .HasForeignKey(a => a.UserID);
+
+            //Client-User
+            modelBuilder.Entity<Client>()
+                .HasOne<User>(c => c.User)
+                .WithMany(u => u.Clients)
```
Comment (ClaireWhitten) [inline ClassLibrary3/Repository/AppDbContext.cs]: .WITHONE(u => u.Clients)

File: ClassLibrary3/Repository/AppDbContext.cs [hunk 4]
```diff
@@ -146,7 +146,116 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
...
+            //OrderOUT-Client
+            modelBuilder.Entity<OrderOUT>()
+                .HasOne<Client>(oo => oo.Client)
+                .WithMany(c => c.OrderOUTs)
+                .HasForeignKey(oo => oo.ClientId);
+
+            //Supplier-User
+            modelBuilder.Entity<Supplier>()
+                .HasOne<User>(s => s.User)
+                .WithMany(u => u.Suppliers)
```
Comment (ClaireWhitten) [inline ClassLibrary3/Repository/AppDbContext.cs]: .WITHONE(u => u.Suppliers)

Comment (medkaouchi) [reply ClassLibrary3/Repository/AppDbContext.cs:227]: juist

**Pull request merged**